test/house_test.rb

```diff
@@ -0,0 +1,70 @@
+
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < Minitest::Test
+
+  def setup
+    @house = House.new("$400000", "123 sugar lane")
+  end
+
+  def test_it_exists
+    assert_instance_of House, @house
+  end
+
+  def test_it_has_a_price
+    assert_equal "$400000", @house.price
```

Comment:
Pay close attention to the spec. This should have been converted to an integer at this point: 

```
pry(main)> house.price
#=> 400000
```